deploy/examples/operator.yaml

```diff
@@ -457,6 +457,11 @@ data:
   # NOTE: the value here serves as an example and needs to be
   # updated with node labels that define domains of interest
   # CSI_TOPOLOGY_DOMAIN_LABELS: "kubernetes.io/hostname,topology.kubernetes.io/zone,topology.rook.io/rack"
+
+  # To indicate the image pull policy to be applied to the csi driver
+  ROOK_CSI_IMAGE_PULL_POLICY: "IfNotPresent"
+  # To indicate the image pull policy to be applied to ceph pods
+  ROOK_CEPH_IMAGE_PULL_POLICY: "IfNotPresent"
```

Comment:
We shouldn't need this var. The behavior I'm thinking is:
- If a ceph image (starting the mon, mgr, osd, etc), read the new setting from the cluster CR. If not set in the cluster CR, default to IfNotPresent.
- If a CSI image, read the setting from the configmap (that you defined on line 462). If the value is not found, default to IfNotPresent. 
```suggestion
```

Reply:
how about we just have one field for both CSI/ceph pods for the pull policy? will the user would like a different pull policy for different pods? if yes then make sense for two fields.

Also, I like this approach of yours https://github.com/rook/rook/issues/10783#issuecomment-1225994662 @travisn

IMO, the field for ImagePullPolicy makes sense better in operatory.yaml. Also, if someone in the future would request these for filesystem or rgw then we'll have to add in there cr.

Reply:
Reviewing this design again today and comparing to my [previous comment](https://github.com/rook/rook/issues/10783#issuecomment-1225994662) on only using an env var, I'm still preferring my suggestion [yesterday](https://github.com/rook/rook/pull/10966/files#r971099467). Wherever the ceph image is being loaded, we will already have the cephcluster `spec.CephVersion` struct available, so it will be simple to add that setting there. And the CSI pull policy can load the settings just like all the other CSI settings in the configmap. Perhaps nobody will ever have a different policy between the cluster and the csi driver, but I prefer it because it just fits the patterns we already have for cluster and CSI settings. The same pattern will also make sense in #10956.

Reply:
@avanthakkar, PTAL on the above approach and make changes accordingly. Thanks!

Reply:
Done!